.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: bash
+sudo: true
```

Comment:
@aplanas This is the correct list see point one in [https://github.com/travis-ci/apt-package-whitelist/blob/master/README.md](README.md).

I don't get the second point. We want set sudo to false but when we do so we need to declare the needed packages in the travis file. Using pip in the Makefile changes nothing because this definitely needs sudo.